torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16983,7 +16983,7 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
     PythonRefInfo(
         "_refs.permute",
         torch_opinfo_name="permute",
-        supports_nvfuser=False,
+        validate_view_consistency=False,
```

Comment:
Why do we need to skip `validate_view_consistency`?

Reply:
Because nvfuser's execute function always returns a copy.

Reply:
But you are disabling a view consistency check for the ref itself, not just nvfuser executor?

Reply:
Yes, I will revert this.